en/README.md

```diff
@@ -0,0 +1,1138 @@
+|fetchId|String255|O|-|This is ID that indicates whether response for request from Hatohol server. Insert fetchId value to this object, in case of only receive fetchEvents request.
+
+***events object***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|eventId     |String255  |M|-|Id of event.|
+|time        |TimeStamp  |M|-|Time of happened event.|
+|type        |string     |M|-|Type of event.[[$B0lMw(B](#user-content-eventtype)]|
+|triggerId   |String255  |O|-|Trigger id of ignition this event. It is not mandatory because can do not to assciate trigger and event.|
+|status      |string     |O|-|Status of trigger. [[$B0lMw(B](#user-content-triggerstatus)]|
+|severity    |string     |O|-|Severity of trigger. [[$B0lMw(B](#user-content-triggerseverity)]|
+|hostId      |String255  |O|-|Host id of happened event.|
+|hostName    |String255  |O|-|Host name of happened event.|
+|brief       |String255  |M|-|Explanation of event. It is shown by on the WebUI.|
+|extendedInfo|String32767|O|-|Can write a additional information.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "fetchId": "1",
+    "lastInfo": "201504011759",
+    "events": [
+      {
+        "extendedInfo": "sampel extended info",
+        "brief": "example brief",
+        "eventId": "1",
+        "time": "20150323151300",
+        "type": "GOOD",
+        "triggerId": 2,
+        "status": "OK",
+        "severity": "INFO",
+        "hostId": 3,
+        "hostName": "exampleName"
+      }
+    ]
+  },
+  "method": "putEvents",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### putHostParents(method)
+
+ When HAP complates to connect to Hatohol server,  send all host parent relations to use "ALL" option to Hatohol server.
+
+In case of use "UPDATE" option, send diffarence of host parent relations that based on lastInfo that got from [getLastInfo](#user-content-getlastinfomethod) or saved in internal to Hatohol server. 
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostParents  |object array|M|-|This is object array that stores host parent relations information. Look at the following table for details.|
+|updateType|string    |M|-|Select sending option that match the situation from [[List](#user-content-updatetype)].|
+|lastInfo    |String32767|O|-|Insert to here host parent relation information of last. This information is result of [getLastInfo](#user-content-getlastinfomethod).|
+
+***hostParents object***
+
+ In case of remove host parent relation from Hatohol server, can remove host parent relation that is to sent childHostId by insert empty string to parentHostId.
```

Comment:
> can remove host parent relation that is to sent childHostId by insert empty string to parentHostId.

API users can remove host parent relation against childHostId with sending a request which contains empty string in parentHostId.